### ISSUE TITLE
Creating a simulator while keeping pretrained weights

### ISSUE THREAD

**Issue by MojoForPresident** [nengo/nengo-dl#202]
Hello everyone!

I am currently trying to convert a Keras MNIST classifier network (with pretrained weights) to native Nengo objects and simulate it using the nengo_dl.Simulator class. My code for that looks as follows:
```python
# The `keras_model` has already been trained using Keras
converter = nengo_dl.Converter(keras_model, allow_fallback=False)
with nengo_dl.Simulator(converter.net, minibatch_size=1) as sim:
    result = sim.predict({converter.inputs[keras_inp]: test_input})
```
When I check if the classification still working and has the same accuracy, the network does not perform as expected but shows accuracies of around 10% (meaning the weights are randomly assigned).
```python
predictions = np.argmax(result[converter.outputs[keras_outp]], axis=-1)
ground_truth = np.argmax(test_target, axis=-1)
# Results in about 0.1
accuracy = (predictions == ground_truth).mean()
```
Now my question is: How can a keep the pretrained weights of the network in the converter when creating a new Simulator instance?

Thanks a lot for your help!
Cheers.

**Comment by xchoo** [comment]
Hi @MojoForPresident! For future reference, we have a dedicated forum (https://forum.nengo.ai/) for questions like these. We tend to keep the Github issues for bugs and such. 😄 

As for your question, the code you have, namely:
```
converter = nengo_dl.Converter(keras_model, allow_fallback=False)
with nengo_dl.Simulator(converter.net, minibatch_size=1) as sim:
   ...
```
Should generate and run a Nengo model with the same weights as the Keras model. Without the complete code to run and test, I would advise you check that the shape of your inputs and outputs are what is expected. 

As an example, if your network is a classification network with 10 output classes (or 10 nodes in the output layer), I would expect the training labels to be a one-hot representation of the correct class, instead of the numerical representation of that class. I.e., if the correct class is 4 (starting at 1), the label should be `[0, 0, 0, 1, 0, 0, 0, 0, 0, 0]` instead of `4`).

**Comment by MojoForPresident** [comment]
Hey and thank you very much for answering that quickly!
Okay yeah I think i kept all that in mind, still no luck. But I will close this issue and ask the question again on the forum!
Sorry for opening an issue for that and thanks again!

**Comment by xchoo** [comment]
When you make the forum post, if you could provide some kind of code that replicates your issue, it would speed up the debugging process.
